lightyear_crossbeam/src/lib.rs

```diff
@@ -39,8 +39,8 @@ const LOCALHOST: SocketAddr = SocketAddr::new(core::net::IpAddr::V4(Ipv4Addr::LO
 /// and receiver ends of the channels.
 #[derive(Component, Clone)]
 #[require(Link::new(None))]
-#[require(LocalAddr(LOCALHOST))]
-#[require(PeerAddr(LOCALHOST))]
+// #[require(LocalAddr(LOCALHOST))]
```

Comment:
I think we might still want those, no?
If the user provides their own LocalAddr and PeerAddr, then these won't be inserted